asset/asset.go

```diff
@@ -0,0 +1,38 @@
+package asset
+
```

Comment:
We can add one line here
```suggestion
//go:generate mockery --name Repository --outpkg mock --output ../lib/mock/ --structname AssetRepository --filename asset_repository.go
```

so we can easily generate mock to `lib/mock`

Reply:
@mabdh I tried adding the line and run `mockery --all`, but it does not seem to have any difference. Did I miss something?

Reply:
Oh for this, you just need to run `go generate` inside the folder. Or in VScode, there should be a click button helper above it.
<img width="1179" alt="Screen Shot 2022-02-02 at 15 09 03" src="https://user-images.githubusercontent.com/4079709/152116320-5d64dbd5-0f42-4101-b108-56e0070dc2a5.png">

Reply:
ah got it, thanks!